layouts/shortcodes/faq_questions.html

```diff
@@ -0,0 +1,4 @@
+<! - Note this is in layouts/partials/faq_questions.html and layouts/shortcodes/faq_questions.html -->
```

Comment:
Why is this in both places?

Also, isn't the `<! -` supposed to be `2 hyphens` not `space-hyphen`?

Reply:
D-ohhh!I I thought I had fixed that.